src/styling/ProjectsPage.css

```diff
@@ -1,101 +1,174 @@
 /** Multiple card view **/
 
-.pg-container {
+.projects-page {
   max-width: 1400px;
+  margin: 3rem auto;
+}
+
+.projects-page .flow-images {
+  margin: 1rem 0;
 }
 
 .preview-card {
-  border-radius: 10px;
-  margin: 20px 0px;
-  padding: 5px;
+  border-radius: 16px;
+  margin: 16px 0px;
+  box-shadow: 8px 8px 0px 2px rgba(249,245,227,1);
+  padding: 4px;
+  cursor: pointer;
+  background-color: var(--main-grey);
 }
 
-.urgent-tag {
-  border-radius: 50px;
+.preview-card:hover {
   background-color: var(--main-yellow);
-  font-weight: bold;
   color: var(--main-black);
-  border: transparent;
 }
 
-.card-row {
-  margin-top: 10px;
+.preview-card:hover h5 {
+  color: var(--main-black);
+}
+
+.preview-subtitle {
+  margin-bottom: 16px;
+  font-size: 20px;
+}
+
+.project-card-urgent {
+  border-radius: 8px;
+  border: 2px solid var(--main-blue);
+  font-family: HelveticaNeueBold;
+  color: var(--main-blue);
+  padding: 4px 8px;
+  margin: 4px 0 16px;
+  font-size: 14px;
+  display: inline-block;
 }
 
-.center-div {
-  margin: 0 auto;
+.project-tag-space {
+  height: 46px;
+  display: inline-block;
 }
 
-.overview {
-  margin: 0 auto;
-  width: 90vw;
+.project-more-link {
+  color: var(--main-blue);
 }
 
-.broad {
-  border-radius: 10px;
-  margin: 15px 0px;
-  padding: 10px;
+.project-card-description {
+  line-height: 1.3rem;
+  height: calc(1.3rem * 5);
+  overflow: hidden;
+  text-overflow: ellipsis;
 }
 
 /** Selected card view **/
```

Comment:
Could we add more padding for the sides of the page and decrease the space in between the two columns? 
<img width="487" alt="Screen Shot 2020-07-16 at 8 19 32 AM" src="https://user-images.githubusercontent.com/43453743/87670223-66cf1000-c73d-11ea-8a56-8e684971cba0.png">